drivers/ipm/CMakeLists.txt

```diff
@@ -6,7 +6,6 @@ zephyr_library()
 
 zephyr_library_sources_ifdef(CONFIG_IPM_MCUX   ipm_mcux.c)
```

Comment:
I think there's a typo in commit message here: "The
driver can now be enabled with _CONFIG_IPM_IMX_REV2_." - s/CONFIG_IPM_IMX_REV2/_CONFIG_IPM_IMX

Reply:
There is, thanks for noticing. I've updated the PR